apps/studio/components/interfaces/Settings/Database/DatabaseSettings/DatabaseSettings.tsx

```diff
@@ -102,135 +102,129 @@ const DatabaseSettings = () => {
+                  poolingMode === 'session' &&
+                  snap.usePoolerConnection && <IPv4AddonDirectConnectionNotice />}
+                {ipv4Addon === undefined && !snap.usePoolerConnection && <IPv4DeprecationNotice />}
+                {isMd5 && (
+                  <Alert_Shadcn_>
+                    <WarningIcon />
+                    <AlertTitle_Shadcn_>
+                      If you are connecting to your database via a GUI client, use the{' '}
+                      <span tabIndex={0}>connection string</span> above instead
+                    </AlertTitle_Shadcn_>
+                    <AlertDescription_Shadcn_>
+                      GUI clients only support database connections for Postgres 13 via a connection
+                      string.
+                    </AlertDescription_Shadcn_>
+                  </Alert_Shadcn_>
+                )}
+              </div>
+              <Input
+                className="input-mono"
+                layout="horizontal"
+                readOnly
+                copy
+                disabled
+                value={connectionInfo.db_host}
+                label="Host"
+                onCopy={() => {
+                  handleCopy('Host')
+                }}
+              />
+              <Input
+                className="input-mono"
+                layout="horizontal"
+                readOnly
+                copy
+                disabled
+                value={connectionInfo.db_name}
+                label="Database name"
+              />
+              <Input
+                className="input-mono"
+                layout="horizontal"
+                readOnly
+                copy
+                disabled
+                value={poolingMode === 'transaction' ? connectionInfo.db_port : '5432'}
+                label="Port"
+              />
+              {isMd5 && snap.usePoolerConnection && (
                 <Input
+                  className="input-mono"
                   layout="horizontal"
-                  className="input-mono table-input-cell text-base"
                   readOnly
                   copy
                   disabled
-                  value={connectionInfo.db_user}
-                  label="User"
-                />
-                <Input
-                  className="input-mono"
-                  layout="horizontal"
-                  disabled
-                  readOnly
-                  value={
-                    state.selectedDatabaseId !== projectRef
-                      ? '[The password for your primary database]'
-                      : '[The password for your database]'
-                  }
-                  label="Password"
+                  value={`reference=${projectRef}`}
+                  label="Options"
                 />
-              </>
-            )}
-          </Panel.Content>
-        </Panel>
-      </section>
-
-      <ResetDbPassword disabled={isLoading || isError} />
```

Comment:
i actually dont see why there was a need to disable this based on the statuses of read replicas, addons, and pooler config - if anyone might remember